integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -179,6 +193,21 @@ object CommonUtil {
     isValid
   }
 
+  def validateTypeConvert(desType: Option[String], value: String): Boolean = {
+    val result = desType match {
+      case Some("IntegerType") => scala.util.Try(value.trim().toInt)
+      case Some("LongType") => scala.util.Try(value.trim().toLong)
+      case Some("FloatType") => scala.util.Try(value.trim().toFloat)
+      case Some("DoubleType") => scala.util.Try(value.trim().toDouble)
+      case Some("ByteType") => scala.util.Try(value.trim().toByte)
+      case Some("ShortType") => scala.util.Try(value.trim().toShort)
+      case Some("DecimalType") => scala.util.Try(value.trim().toDouble)
```

Comment:
time stamp and date are not validated. 
Go through the complete list of supported data types and handle accordingly